src/main/java/com/conveyal/datatools/manager/controllers/api/DeploymentController.java

```diff
@@ -482,92 +482,48 @@ private static String deploy (Request req, Response res) {
-        // Check parameters supplied in request for validity.
-        Auth0UserProfile userProfile = req.attribute("user");
-        Deployment deployment = getDeploymentWithPermissions(req, res);
-        Project project = Persistence.projects.getById(deployment.projectId);
-        if (project == null) {
-            logMessageAndHalt(req, 400, "Internal reference error. Deployment's project ID is invalid");
-        }
-
-        // Get log upload URI from deploy job
-        AmazonS3URI logUploadS3URI = deployJob.getS3FolderURI();
-
-        // Execute the pelias update job and keep track of it
-        PeliasUpdateJob peliasUpdateJob = new PeliasUpdateJob(userProfile, "Updating Custom Geocoder Database", deployment, logUploadS3URI);
-        JobUtils.heavyExecutor.execute(peliasUpdateJob);
-        return SparkUtils.formatJobMessage(peliasUpdateJob.jobId, "Pelias update initiating.");
-    }
-
     /**
      * Uploads a file from Spark request object to the s3 bucket of the deployment the Pelias Update Job is associated with.
      * Follows https://github.com/ibi-group/datatools-server/blob/dev/src/main/java/com/conveyal/datatools/editor/controllers/api/EditorController.java#L111
      * @return      S3 URL the file has been uploaded to
      */
-    private static String uploadToS3 (Request req, Response res) throws IOException {
+    private static Deployment uploadToS3 (Request req, Response res) {
         // Check parameters supplied in request for validity.
-        Auth0UserProfile userProfile = req.attribute("user");
         Deployment deployment = getDeploymentWithPermissions(req, res);
 
         String url;
-        Exception failure = null;
-
-        // Get file from request
-        if (req.raw().getAttribute("org.eclipse.jetty.multipartConfig") == null) {
-            MultipartConfigElement multipartConfigElement = new MultipartConfigElement(System.getProperty("java.io.tmpdir"));
-            req.raw().setAttribute("org.eclipse.jetty.multipartConfig", multipartConfigElement);
-        }
-        String extension = null;
-        File tempFile = null;
         try {
-            Part part = req.raw().getPart("file");
-            extension = "." + part.getContentType().split("/", 0)[1];
-            tempFile = File.createTempFile(part.getName() + "_csv_upload", extension);
-            InputStream inputStream;
-            inputStream = part.getInputStream();
-            FileOutputStream out = new FileOutputStream(tempFile);
-            IOUtils.copy(inputStream, out);
 
             String keyName = String.join(
                     "/",
                     bundlePrefix,
                     deployment.projectId,
                     deployment.id,
                     // Where filenames are generated. Prepend random UUID to prevent overwriting
-                    UUID.randomUUID() + "_" + part.getSubmittedFileName()
+                    UUID.randomUUID() + ""
```

Comment:
Not too fond of this approach to get a string value. I think I'd prefer 
```suggestion
                    UUID.randomUUID().toString()
```

Reply:
Fixed in 4b73157ca781655923e944dad2ed1ccb0c32ce05